package.json

```diff
@@ -50,14 +50,12 @@
   "spm": {
     "main": "d3.js"
   },
-  "dependencies": {
-    "jsdom": "1.0.0"
-  },
   "devDependencies": {
-    "smash": "~0.0.12",
-    "uglify-js": "2.4.0",
-    "vows": "0.7.0",
-    "seedrandom": "2.3.1"
+    "jsdom": "3",
+    "seedrandom": "2",
+    "smash": "0.0",
+    "uglify-js": "2.4",
```

Comment:
Sure, I’ll restore a specific version for UglifyJS. Unpinning doesn’t really buy us anything since you still have to remember to `npm update` to grab the newer version.